scripts/record_books.py

```
@@ -65,25 +72,278 @@
+    conn.commit()
+    return conn
 
 
-def book(clob_host: str, token_id: str) -> dict:
-    """One book via the shared parse: bad levels are skipped, never raised,
-    so one garbage row cannot abort a recording run mid-window."""
-    r = requests.get(f"{clob_host}/book", params={"token_id": token_id}, timeout=10)
+def book(clob_host: str, token_id: str) -> tuple[dict, float, float]:
+    url = f"{clob_host}/book"
+    t_sent = time.time()
+    r = _SESSION.get(url, params={"token_id": token_id}, timeout=MARKET_TIMEOUT)
```

Ruff (0.16.1)

[error] Undefined name `_SESSION`

(F821)